vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -155,7 +135,6 @@ $global:map = @{
         $defaultContainerdPackageUrl
     );
     "c:\akse-cache\csi-proxy\"    = @(
-        "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz",
```

Comment:
keep this version

Reply:
The version in RP is v1.0.2 for two months. why do we still need to keep this one?

Reply:
ok, let's remove this.